.github/workflows/release.yml

```diff
@@ -0,0 +1,45 @@
+name: Tagged Release
+on:
+  push:
+    tags: ['*']
+
+jobs:
+  build:
+    name: Build and Create Release
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Fetch all tags
+      run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+    - name: Get current tag
+      run: echo "::set-env name=CURRENT_TAG::$(git describe --abbrev=0 --tags $(git rev-list --tags --max-count=1))"
+    - name: Get previous tag
+      run: echo "::set-env name=PREVIOUS_TAG::$(git describe --abbrev=0 --tags $(git rev-list --tags --skip=1 --max-count=1))"
+    - name: Install dependencies
+      run: dotnet restore
+    - name: Build
+      run: dotnet build osu.Game.Rulesets.Rush --configuration Release -p:version=${{env.CURRENT_TAG}} --no-restore
+    - name: Create Release
+      id: create_release
+      uses: actions/create-release@latest
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        tag_name: ${{ github.ref }}
+        release_name: ${{ github.ref }}
+    - name: Upload Release Asset
+      id: upload-release-asset
+      uses: actions/upload-release-asset@master
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      with:
+        upload_url: ${{ steps.create_release.outputs.upload_url }}
+        asset_path: ./osu.Game.Rulesets.Rush/bin/Release/netcoreapp3.1/osu.Game.Rulesets.Rush.dll
```

Comment:
Oh, just noticed, but you should change `netcoreapp3.1` to `netstandard2.1` as your main project doesn't output to that folder.

Reply:
Thanks, I'll fix it up tomorrow.